spec/models/commute_spec.rb

```diff
@@ -8,15 +8,13 @@
       it { should have_valid(:ending_location).when("Attleboro Station") }
       it { should_not have_valid(:ending_location).when(nil, "")}
 
-      it { should have_valid(:time).when("Early Morning", "Morning", "Noon", "Afternoon", "Evening", "Night") }
+      it { should have_valid(:time).when("Early Morning", "Morning", "Afternoon", "Early Evening", "Evening", "Night") }
       it { should_not have_valid(:time).when(nil, "")}
```

Comment:
It would also be worth adding a test here for something that is NOT nil or an empty string, but instead a string that doesn't match your selected times